addons/website/models/ir_ui_view.py

```diff
@@ -293,6 +295,11 @@ def customize_template_get(self, cr, uid, key, full=False, bundles=False, contex
                 })
         return result
 
+    def toggle_options(self, cr, uid, ids, context=None):
+        view_ids = self.search(cr, uid, [('inherit_id', 'child_of', ids), ('active', '=', True)], context=context)
```

Comment:
If a user unactive a parent template (like the footer), all inherit are unactivate but if this template haven't customize_show, the user can't re-activate the templates.
Please fix the domain